### PR TITLE
moved index, ranking to templates folder

### LINKED ISSUE #96: Move rankings html to template folder
The `Rankings.html` file is supposed to be in the `templates` folder and the server has to use it from there.

related to #62

### PR DESCRIPTION
**Pull request by rishabhKalakoti**
fixes #96

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def changePath():
 @app.get("/question/<number>")
 def question(number):
     statement = questions[number].statement
-    return template("index.html", question_number=number, question=statement)
+    return template("templates/index.html", question_number=number, question=statement)
```
Comment (theSage21) [inline server.py]: From what I can [see here](https://bottlepy.org/docs/dev/_modules/bottle.html#template) I think this works without the templates prefix? Can you confirm that the templates prefix is needed? If not let's remove this change

Comment (rishabhKalakoti) [reply server.py]: Done. I checked, it. The folder name was `views` in bottle and `templates` in Flask.
I earlier tested with `templates` name and it did not work so I made the commit.

**Comment by rishabhKalakoti** [comment]
The links to css were `static/css/<filename>` in `index.html` and they are served through the `static_file` function, so there were no links to `css/index.css` and it was redundant to `static/index.css`. That's why I removed it.

**Comment by theSage21** [comment]
oh. I didn't realize that.

**Pull request merged**